src/game/claw.rs

```diff
@@ -39,51 +39,32 @@ impl Claw {
     }
 
     pub fn update(&mut self, parent_loc: Point2) -> GameResult<&Self> {
-        self.location = parent_loc;
+        /*
```

Comment:
Maybe it makes sense to put an input block on points and vectors in here. The thinking (my thinking?) behind the logic of the body anchor and joint anchor is probably not clear right away. I can write a paragraph or two about that

Reply:
I'd love that, thank you! That would help me too 😄